test/test_ops.py

```diff
@@ -1930,7 +1930,7 @@ def _test_fake_crossref_helper(self, device, dtype, op, context):
 
             # TODO: enable check_aliasing, batch norm fails
             with torch._subclasses.CrossRefFakeMode(ignore_op_fn=lambda fn: fn in common_skip_ops, check_aliasing=True):
-                with warnings.catch_warnings(), context():
+                with warnings.catch_warnings(), context(), torch.autograd.set_multithreading_enabled(False):
```

Comment:
this is set in the tests to prevent test failures - is it also set somewhere directly in dynamo?

Reply:
Yes, see https://github.com/pytorch/pytorch/pull/86245